### DISCUSSION TITLE
Mainboard fan noise when heating

### DISCUSSION THREAD

**Discussion by 1driver** [Jyers/Marlin#686] [Q&A] [answered]
Hi there!

I just install E3V2-BLTouch-5x5-HS-v4.2.2 in my Ender 3 v2. Everything is working great, I got the best first layer I've ever had, and the erratic behavior of the bltouch is completely gone; however, I noticed that the mainboard fan now is considerably noisier when the heating of the hotend is active.  Does anyone experience something similar?

Cheers!

**Comment by gothcha** [comment, marked as answer]
The mainboard fan is wired in parallel with the part cooling fan, so maybe the part fan speed settings are different to what you were used to? The fan speed is shown on the display. Settings can also be made in Cura for initial layers and higher layers.
